Gerik/Crypters/Coder/Coder.java

```diff
@@ -1,190 +1,158 @@
+
+
 public class Coder
 {
-    public static void Info() //show helpful info
-    {
+    public static void info(){ //show helpful info
         System.out.print("Usage: $ Java Coder [OPTION] <Text> <KeyWord>\n\nProgram for encrypting and decrypting text by building a matrix and changing the location of columns by using the key word\n\n\t-h\tHelp\n\t-c\tUse for encrypting text\n\t-d\tUse for decrypting text\n\nKeyWord is optional\n\nExamples:\n$ java Coder -c TryMeBitch\n$ java Coder -d SbsaZuayYaklaae Allah\n");
     }
 
-    public static int FindWAndH(int length, boolean Height) //find a width or a height of array
-    {
-        while (true)
-        {
-            for(int i = 10; i > 2; i--)
-            {
-                if(length%i == 0 && length/i>1)
-                {
-                    if(Height){return length/i;}else{return i;}
+    public static int findWidthAndHeight(int length, boolean Height){ //find a width or a height of array
```

Comment:
if you wanna describe what's going on there, use Javadoc
```
/**  find a width or a height of array
 * @param length - blablabla
 * @param Heigth - blablabal
 **/
public static void findWidthAndHeigth(
```
2) why static?
3) non-compilant, as method doesnt return anything
4) Heigth boolean???
5) and please, make two methods - findWidth, and find Heigth